src/composite.c

```diff
@@ -0,0 +1,119 @@
+#include <stdio.h>
+#include <vips/vips.h>
+
+/* Composite images `src` and `dst`.
+ */
+int Composite(VipsObject *context, VipsImage *src, VipsImage *dst,
```

Comment:
> The Google C++ guidelines use PascalCase for (non-accessor) method names, but I suspect that may have been influenced by the prevalence of C++ for Windows development in the 90s.

Thanks for sharing, I didn’t know that. C# uses the same convention.



> The only conclusion I've been able to reach is that consistency within a code base is more important than consistency across code bases.

I wholeheartedly agree and tried to match my new code as closely as possible :+1: